universal-application-tool-0.0.1/app/auth/AdOidcClient.java

```diff
@@ -0,0 +1,9 @@
+package auth;
```

Comment:
suggestion (non-blocking): If you'd like, these annotations can be added directly in the module (in case you don't want separate classes), or could be combined into one `AuthAnnotations` class

Reply:
Oh, that would be a little cleaner.  I think I'm gonna leave it for the moment but if we add any more I'll do that.